settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
Those are both fine points. Is there any way to accomplish them without using the funny project name?